### PR TITLE
Add pause unpause test

### PR DESCRIPTION
**Pull request by chuanchang**
vendor: update protocol
feature: add pause and unpause remote interface
hack: support pause and unpause feature
test: add testing for pause and unpause feature

Test Result
```shell
Running Suite: CRI validation
=============================
Random Seed: 1547802947 - Will randomize all specs
Will run 1 of 82 specs

SSSSSSSS
------------------------------
[k8s.io] Container runtime should support basic operations on container
  runtime should support pause running container [Conformance]
  /root/Workspace/go/src/github.com/kubernetes-sigs/cri-tools/pkg/validate/container.go:102
[BeforeEach] [k8s.io] Container
  /root/Workspace/go/src/github.com/kubernetes-sigs/cri-tools/pkg/framework/framework.go:51
[BeforeEach] [k8s.io] Container
  /root/Workspace/go/src/github.com/kubernetes-sigs/cri-tools/pkg/validate/container.go:65
[BeforeEach] runtime should support basic operations on container
  /root/Workspace/go/src/github.com/kubernetes-sigs/cri-tools/pkg/validate/container.go:74
[It] runtime should support pause running container [Conformance]
  /root/Workspace/go/src/github.com/kubernetes-sigs/cri-tools/pkg/validate/container.go:102
STEP: create container
STEP: Get image status for image: busybox:1.28
STEP: Create container.
Jan 18 17:15:47.946: INFO: Created container "c606fd9ac40db1ca0c3cf853e599e5e4cccbb8c135f27d48b57533734a81e6f2"

STEP: start container
STEP: Start container for containerID: c606fd9ac40db1ca0c3cf853e599e5e4cccbb8c135f27d48b57533734a81e6f2
Jan 18 17:15:48.067: INFO: Started container "c606fd9ac40db1ca0c3cf853e599e5e4cccbb8c135f27d48b57533734a81e6f2"

STEP: Get container status for containerID: c606fd9ac40db1ca0c3cf853e599e5e4cccbb8c135f27d48b57533734a81e6f2
Jan 18 17:15:48.068: INFO: the current state of container: CONTAINER_RUNNING
STEP: pause container
STEP: test pasue running container
STEP: Get container status for containerID: c606fd9ac40db1ca0c3cf853e599e5e4cccbb8c135f27d48b57533734a81e6f2
Jan 18 17:15:48.084: INFO: the state of container after pausing: CONTAINER_CREATED
STEP: unpause container
STEP: test recover pasued container
STEP: Get container status for containerID: c606fd9ac40db1ca0c3cf853e599e5e4cccbb8c135f27d48b57533734a81e6f2
Jan 18 17:15:48.100: INFO: the current state of container after unpausing: CONTAINER_RUNNING
STEP: test stop container
STEP: Stop container for containerID: c606fd9ac40db1ca0c3cf853e599e5e4cccbb8c135f27d48b57533734a81e6f2
Jan 18 17:15:48.218: INFO: Stopped container "c606fd9ac40db1ca0c3cf853e599e5e4cccbb8c135f27d48b57533734a81e6f2"

STEP: Get container status for containerID: c606fd9ac40db1ca0c3cf853e599e5e4cccbb8c135f27d48b57533734a81e6f2
[AfterEach] runtime should support basic operations on container
  /root/Workspace/go/src/github.com/kubernetes-sigs/cri-tools/pkg/validate/container.go:78
STEP: stop PodSandbox
STEP: delete PodSandbox
[AfterEach] [k8s.io] Container
  /root/Workspace/go/src/github.com/kubernetes-sigs/cri-tools/pkg/framework/framework.go:52
•SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
Ran 1 of 82 Specs in 1.061 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 81 Skipped PASS
```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,12 @@
...
+dist: xenial
 
 go:
   - "1.10.2"
 
 go_import_path: github.com/kubernetes-sigs/cri-tools
 
-before_install:
-  - sudo env PATH=$PATH GOPATH=$GOPATH hack/install-cni.sh
-  - sudo env PATH=$PATH GOPATH=$GOPATH hack/install-pouch.sh
```
Comment (zhuangqh) [inline .travis.yml]: This step will run before each stage. Some stage don't need it.

File: hack/run-pouch.sh [hunk 2]
```diff
@@ -51,36 +51,32 @@ pouch::check_cri_listening() {
 
 # pouchd::install_dependencies downloads and installs dependent packages
 pouch::install_dependencies() {
+  sudo apt-get update
+  sudo apt-get install -y socat
```
Comment (zhuangqh) [inline hack/run-pouch.sh]: dependency of CRI stream server

File: pkg/validate/container.go [hunk 3]
```diff
@@ -99,6 +99,38 @@ var _ = framework.KubeDescribe("Container", func() {
...
+			By("create container")
+			containerID := framework.CreateDefaultContainer(rc, ic, podID, podConfig, "container-for-pause-unpause-test-")
+
+			By("start container")
+			startContainer(rc, containerID)
+
+			By("get container status")
+			Eventually(func() runtimeapi.ContainerState {
+				return getContainerStatus(rc, containerID).State
+			}, time.Minute, time.Second*4).Should(Equal(runtimeapi.ContainerState_CONTAINER_RUNNING))
```
Comment (zhuangqh) [inline pkg/validate/container.go]: Wait a while for the container ready and state transform.

File: .travis.yml (hunk 1 above)
Comment (chuanchang) [reply .travis.yml]: 👌

File: hack/run-pouch.sh (hunk 2 above)
Comment (chuanchang) [reply hack/run-pouch.sh]: LGTM

File: pkg/validate/container.go (hunk 3 above)
Comment (chuanchang) [reply pkg/validate/container.go]: LGTM

**Comment by zhuangqh** [comment]
thanks @chuanchang  merging...

**Pull request merged**